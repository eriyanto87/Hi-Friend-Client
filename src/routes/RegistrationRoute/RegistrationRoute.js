import React from "react";
import Registration from "../../components/Registration/Registration";

export default function RegistrationRoute() {
  return (
    <>
      <Registration />
    </>
  );
}
