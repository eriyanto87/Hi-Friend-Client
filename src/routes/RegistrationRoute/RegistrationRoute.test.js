import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import RegistrationRoute from "./RegistrationRoute";

describe("Registration Route", () => {
  it("Registration Snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <RegistrationRoute />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <RegistrationRoute />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
