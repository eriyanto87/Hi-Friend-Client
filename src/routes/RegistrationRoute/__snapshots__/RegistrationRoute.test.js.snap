// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration Route Registration Snapshot 1`] = `
Array [
  <div
    className="home"
  >
    <div>
      <a
        href="/"
      >
        Go To Home
      </a>
    </div>
  </div>,
  <div
    className="registration-container"
  >
    <section
      className="registration-header"
    >
      <h1>
        Register
      </h1>
      <p>
        Create your account. It's free and only takes a minute!
      </p>
    </section>
    <form
      onSubmit={[Function]}
    >
      <section
        className="name"
      >
        <p>
          <input
            name="first"
            placeholder="First Name"
            required={true}
            type="text"
          />
        </p>
        <p>
          <input
            name="last"
            placeholder="Last Name"
            required={true}
            type="text"
          />
        </p>
      </section>
      <section
        className="registration-info"
      >
        <p>
          <input
            name="username"
            placeholder="Username"
            required={true}
            type="text"
          />
        </p>
        <p>
          <input
            name="password"
            placeholder="Password"
            required={true}
            type="password"
          />
        </p>
      </section>
      <button>
        Register Now
      </button>
    </form>
  </div>,
  <p
    className="signin-section"
  >
    Already have an account? 
    <a
      href="/signin"
    >
      Sign In
    </a>
  </p>,
  <div
    className="footer"
  >
    Â© 2020
  </div>,
]
`;
